Mathlib/Analysis/InnerProductSpace/JointEigenspace.lean

```diff
@@ -35,22 +35,15 @@ symmetric operator, simultaneous eigenspaces, joint eigenspaces
 
 -/
 
-variable {𝕜 E : Type*} [RCLike 𝕜]
-variable [NormedAddCommGroup E] [InnerProductSpace 𝕜 E]
-
 open Module.End
 
 namespace LinearMap
 
 namespace IsSymmetric
 
-@[simp]
-theorem Module.End.genEigenspace_one {R M : Type*} [CommRing R] [AddCommGroup M] [Module R M]
-    (f : Module.End R M) (μ : R) : (f.genEigenspace μ) 1 = f.eigenspace μ :=
-  rfl
-
-section Pair
+section CommRing
 
+variable {𝕜 E n m: Type*} [CommRing 𝕜] [AddCommGroup E] [Module 𝕜 E]
```

Comment:
```suggestion
variable {𝕜 E n m : Type*} [CommRing 𝕜] [AddCommGroup E] [Module 𝕜 E]
```

Reply:
I would put the type variables outside of the `CommRing` section, especially because they are all restated in the next section.

Reply:
Cool. Was wondering about that.